### DISCUSSION TITLE
read from url

### DISCUSSION THREAD

**Discussion by MAGALLANESJoseManuel** [networkx/networkx#7051] [Q&A] [answered]
how may I read a GRAPHML file from a github URL into networkx
This is the address:
https://github.com/SocialAnalytics-StrategicIntelligence/codes/raw/main/graphdata/EliteNet.graphml

**Comment by MridulS** [comment, marked as answer]
You can download the file and read the graphml file from your disk with networkx.

**Reply by MAGALLANESJoseManuel** [reply]
I meant, directly from the URL, as you can do with other  files when reading into pandas, for example.

**Reply by MridulS** [reply]
We don't support that.
